target/linux/rockchip/armv8/base-files/etc/hotplug.d/net/40-net-smp-affinity

```diff
@@ -29,6 +29,7 @@ set_interface_core() {
 }
 
 case "$(board_name)" in
+friendlyarm,nanopc-t6|\
```

Comment:
sorry if I made any confusing, it should be like this:
```
friendlyarm,nanopc-t6|\
friendlyarm,nanopi-r5c|\
radxa,e25|\
sinovoip,rk3568-bpi-r2pro)
	set_interface_core 2 "eth0"
	set_interface_core 4 "eth1"
	;;
friendlyarm,nanopi-r2c|\
friendlyarm,nanopi-r2c-plus|\
friendlyarm,nanopi-r2s|\
radxa,cm3-io|\
xunlong,orangepi-r1-plus|\
xunlong,orangepi-r1-plus-lts)
	set_interface_core 2 "eth0"
	set_interface_core 4 "eth1" "xhci-hcd:usb[0-9]+"
	;;
[...]
```

and the network part:
```
	friendlyarm,nanopc-t6|\
	friendlyarm,nanopi-r5c|\
	radxa,e25)
		ucidef_set_interfaces_lan_wan 'eth0' 'eth1'
		;;
	friendlyarm,nanopi-r2c|\
	friendlyarm,nanopi-r2c-plus|\
	friendlyarm,nanopi-r2s|\
	friendlyarm,nanopi-r4s|\
	friendlyarm,nanopi-r4s-enterprise|\
	radxa,rockpi-e|\
	xunlong,orangepi-r1-plus|\
	xunlong,orangepi-r1-plus-lts)
		ucidef_set_interfaces_lan_wan 'eth1' 'eth0'
		;;
[...]
```

Reply:
Thank you for the example code.